AFQ/api/bundle_dict.py

```diff
@@ -740,3 +740,57 @@ def load_templates(self):
         # reuse probability map from ILF
         self.templates["MdLF_L_prob_map"] = self.templates["ILF_L_prob_map"]
         self.templates["MdLF_R_prob_map"] = self.templates["ILF_R_prob_map"]
+
+    def _gen(self, bundle_name):
+        if bundle_name == "OR_L":
+            self._dict["OR_L"] = {
+                "include": [self.templates["OR_left_roi3"]],
+                "start": self.templates["OR_leftThal"],
+                "end": self.templates["OR_leftV1"],
+                "cross_midline": False
+            }
+        elif bundle_name == "OR_R":
+            self._dict["OR_R"] = {
+                "include": [self.templates["OR_right_roi3"]],
+                "start": self.templates["OR_rightThal"],
+                "end": self.templates["OR_rightV1"],
+                "cross_midline": False
+            }
+        elif bundle_name == "pARC_L":
```

Comment:
Do we want to add at least pARC and VOF to the default set that always gets run?

Reply:
This might be something to ask Mareike, but I think this is fine for now